clients/wallet/src/stellar_wallet.rs

```diff
@@ -494,6 +582,152 @@ mod test {
 		wallet.read().await.cache.remove_dir();
 	}
 
+	#[tokio::test]
+	#[serial]
+	async fn sending_payment_using_claimable_balance_works() {
+		let wallet = wallet("resources/sending_payment_using_claimable_balance_works").clone();
+		let mut wallet = wallet.write().await;
+
+		// let's cleanup, just to make sure.
+		wallet.cache.remove_all_tx_envelopes();
+
+		let destination =
+			PublicKey::from_encoding("GA5ZSEJYB37JRC5AVCIA5MOP4RHTM335X2KGX3IHOJAPP5RE34K4KZVN")
+				.expect("should return a public key");
+
+		let asset = CurrencyId::try_from((
+			"USDC",
+			"GAKNDFRRWA3RPWNLTI3G4EBSD3RGNZZOY5WKWYMQ6CQTG3KIEKPYWAYC",
```

Comment:
Don't we have a constant for this exact USDC issuer already somewhere? Maybe we can re-use it somehow?